.gitlab-ci.yml

```diff
@@ -63,11 +61,26 @@ build:testing:
     paths:
       - testingImage.tar
 
+build:qa:
+  stage: build
+  tags:
+    - mender-qa-slave-highcpu
```

Comment:
Do we need this for dind to work? Otherwise I would suggest removing it since both apt and pip are single threaded, so additional cores will not speed them up.

Reply:
the installation step for `linux-modules` here https://github.com/mendersoftware/mender-test-containers/pull/26/files#diff-6e88c68cd8f590f111ddff8b5a219bc7d3954873c05eced36f24795e20708dbdR11 needs to happen on the same machine type to get the proper modules...